ext/dapr-ext-workflow/dapr/ext/workflow/workflow_runtime.py

```diff
@@ -80,3 +107,141 @@ def start(self):
     def shutdown(self):
         """Stops the listening for work items on a background thread."""
         self.__worker.stop()
+
+    def workflow(self, __fn: Workflow = None, *, name: Optional[str] = None):
+        """Decorator to register a workflow function.
+
+        This example shows how to register a workflow function with a name:
+
+                from dapr.ext.workflow import WorkflowRuntime
+                wfr = WorkflowRuntime()
+
+                @wfr.workflow(name="add")
+                def add(ctx, x: int, y: int) -> int:
+                    return x + y
+
+        This example shows how to register a workflow function without a name:
```

Comment:
It's not that it registers it without a name, but that it uses the function name as the workflow name.